server/src/main/java/io/seata/server/controller/HealthController.java

```diff
@@ -24,6 +24,7 @@
 /**
  * @author spilledyear@outlook.com
  */
+@Deprecated
```

Comment:
Why is it marked as deprecated?

Reply:
the health check url change to 'http://127.0.0.1:7091/actuator/health'

Reply:
Fixed